### PR TITLE
Beautifier losses doc

### PR DESCRIPTION
**Pull request by WindQAQ**
# Description

Beautifier losses doc. Our doc is very messy 😢.
- Encourage to use backticks to wrap the shape. https://www.tensorflow.org/community/contribute/docs_style#tensors
- For `tf.*`, one should wrap it with backticks so that the hyperlink can be generated. https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown
- Encourage to use mathjax syntax for math expr. https://www.tensorflow.org/community/contribute/docs_style#math_in_markdown

https://www.tensorflow.org/addons/api_docs/python/tfa/losses

## Type of change

- [ ] Bug fix
- [ ] New Tutorial
- [x] Updated or additional documentation
- [ ] Additional Testing
- [ ] New Activation and the changes conform to the [activation contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/activations/README.md#contribution-guidelines)
- [ ] New Callback and the changes conform to the [callback contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/callbacks/README.md#contribution-guidelines)
- [ ] New Image addition and the changes conform to the [image op contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/README.md#contribution-guidelines)
- [ ] New Layer and the changes conform to the [layer contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/README.md#contribution-guidelines)
- [ ] New Loss and the changes conform to the [loss contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/losses/README.md#contribution-guidelines)
- [ ] New Metric and the changes conform to the [metric contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/metrics/README.md#contribution-guidelines)
- [ ] New Optimizer and the changes conform to the [optimizer contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/optimizers/README.md#contribution-guidelines)
- [ ] New RNN Cell and the changes conform to the [rnn contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/rnn/README.md#contribution-guidelines)
- [ ] New Seq2seq addition and the changes conform to the [seq2seq contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/seq2seq/README.md#contribution-guidelines)
- [ ] New Text addition and the changes conform to the [text op contribution guidelines](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/text/README.md#contribution-guidelines)

# Checklist:

- [x] I've properly [formatted my code according to the guidelines](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#coding-style)
    - [ ] By running Black + Flake8
    - [x] By running pre-commit hooks
- [ ] This PR addresses an already submitted issue for TensorFlow Addons
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This PR contains modifications to C++ custom-ops

# How Has This Been Tested?

If you're adding a bugfix or new feature please describe the tests that you ran to verify your changes:
*

### CODE REVIEW

File: tensorflow_addons/losses/triplet.py [hunk 1]
```diff
@@ -77,30 +77,21 @@ def triplet_semihard_loss(
...
       margin: Float, margin term in the loss definition.
-      distance_metric: str or function, determines distance metric:
-                       "L2" for l2-norm distance
-                       "squared-L2" for squared l2-norm distance
-                       "angular" for cosine similarity
-                        A custom function returning a 2d adjacency
-                          matrix of a chosen distance metric can
-                          also be passed here. e.g.
-
-                          def custom_distance(batch):
```
Comment (lc0) [inline tensorflow_addons/losses/triplet.py:93]: would make sense to format this example as python example, or?

**Comment by bhack** [comment]
Did you use any tools?

File: tensorflow_addons/losses/triplet.py [hunk 2]
```diff
@@ -77,30 +77,21 @@ def triplet_semihard_loss(
...
-                              batch = 1 - batch @ batch.T
-                              return batch
-
-                          triplet_semihard_loss(batch, labels,
-                                        distance_metric=custom_distance
-                                    )
+      distance_metric: `str` or a callable that determines distance metric.
```
Comment (lc0) [inline tensorflow_addons/losses/triplet.py:85]: we highlight `str` but not `Callable`. Does it make sense to keep it consistent?

**Comment by WindQAQ** [comment]
> Did you use any tools?

No, I just use my eyes and build doc locally to check the result.

https://github.com/tensorflow/addons/tree/master/docs

File: tensorflow_addons/losses/triplet.py [hunk 3]
```diff
@@ -26,8 +26,8 @@ def _masked_maximum(data, mask, dim=1):
     """Computes the axis wise maximum over chosen elements.
 
     Args:
-      data: 2-D float `Tensor` of size [n, m].
-      mask: 2-D Boolean `Tensor` of size [n, m].
+      data: 2-D float `Tensor` of size `[n, m]`.
```
Comment (lc0) [inline tensorflow_addons/losses/triplet.py:29]: I guess it's better to switch to `with shape`

File: tensorflow_addons/losses/triplet.py (hunk 1 above)
Comment (WindQAQ) [reply tensorflow_addons/losses/triplet.py:93]: seems that is impossible to make a code block under listing env

File: tensorflow_addons/losses/triplet.py (hunk 2 above)
Comment (WindQAQ) [reply tensorflow_addons/losses/triplet.py:85]: Good catch!

File: tensorflow_addons/losses/triplet.py (hunk 1 above)
Comment (lc0) [reply tensorflow_addons/losses/triplet.py:93]: ah, we could move it from params section and just move to general doc string examples

**Comment by lc0** [comment]
Thanks for making the docs more beautiful! 🥳

**Comment by AakashKumarNain** [comment]
Thanks for doing this @WindQAQ  This is really good

File: tensorflow_addons/losses/focal_loss.py [hunk 4]
```diff
@@ -38,15 +38,11 @@ class SigmoidFocalCrossEntropy(LossFunctionWrapper):
...
-    fl = tfa.losses.SigmoidFocalCrossEntropy()
-    loss = fl(
-      y_true = [[1.0], [1.0], [0.0]],
-      y_pred = [[0.97], [0.91], [0.03]])
-    print('Loss: ', loss.numpy())  # Loss: [6.8532745e-06,
-                                            1.9097870e-04,
-                                            2.0559824e-05]
-    ```
+    >>> fl = tfa.losses.SigmoidFocalCrossEntropy()
+    >>> loss = fl([[0.97], [0.91], [0.03]], [[1.0], [1.0], [0.0]])
```
Comment (AakashKumarNain) [inline tensorflow_addons/losses/focal_loss.py]: I think you swapped the order of `y_true` and `y_pred` here

Comment (WindQAQ) [reply tensorflow_addons/losses/focal_loss.py]: That's from #667. I am going to revert it as it is incomplete.
